### PR TITLE
chore(about): Update to freeCodeCamp

### PR DESCRIPTION
**Pull request by raisedadead**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -89,8 +89,8 @@
 						<div class="palette-pad">
 							<h4>Primary Color</h4>
 							<span class="small">
-								<a href="downloads/freeCodeCamp.png" download>PNG</a> | 
-							<a href="downloads/freeCodeCamp.jpg" download>JPEG</a> | 
+								<a href="downloads/freeCodeCamp.png" download>PNG</a> |
+							<a href="downloads/freeCodeCamp.jpg" download>JPEG</a> |
```
Comment (ghost) [inline index.html:93]: While you're at it, can you fix the indentation issues here?

**Review by ghost — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (raisedadead) [reply index.html:93]: Will make a separate PR?

**Review by systimotic — APPROVED** [review]
LGTM 👍
@atjonathan can you merge?

File: index.html (hunk 1 above)
Comment (ghost) [reply index.html:93]: Sure

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
Thanks @raisedadead 🎉 👍

**Pull request merged**